### PR TITLE
Adaptive breadcrumb

### PR DESCRIPTION
**Pull request by magdalenaan**
related https://github.com/telerik/kendo-themes-private/issues/96

### CODE REVIEW

File: packages/fluent/scss/breadcrumb/_variables.scss [hunk 1]
```diff
@@ -159,13 +202,62 @@ $kendo-breadcrumb-current-item-bg: null !default;
...
+/// The horizontal padding of the medium Breadcrumb link icon.
 /// @group breadcrumb
-$kendo-breadcrumb-icon-link-padding-y: $kendo-breadcrumb-icon-link-padding-x !default;
+$kendo-breadcrumb-md-icon-link-padding-x: $kendo-breadcrumb-icon-link-padding-x !default;
+/// The horizontal padding of the large Breadcrumb link icon.
+/// @group breadcrumb
+$kendo-breadcrumb-lg-icon-link-padding-x: $kendo-breadcrumb-lg-icon-link-padding-y !default;
 
 // Root link spacing
 $kendo-breadcrumb-root-link-spacing: 0px !default;
```
Comment (inikolova) [inline packages/fluent/scss/breadcrumb/_variables.scss:232]: I believe we could remove this var, as it is not necessary any more. According to the updated design, there is not padding set to the root item

**Comment by inikolova** [comment]
Great work, Magi :) Just one more note - in some of the documented variables the first letter of the Breadcrumb is uppercase, in other it is lowercase (mostly in the Fluent theme). Could you please make all of them uppercase?

Note: I checked with the designers team the changed height of the Breadcrumb (medium size) in the Fluent theme and confirm it is intentional.

**Comment by dmanova** [comment]
> Breadcrumb adaptive rendering -> Is the rendering in the breadcrumb-adaptive.tsx the final one for adaptiveness (it looks pretty the same as the other tests)? Wouldn't the Breadcrumb be rendered inside an ActionSheet ([link](https://www.figma.com/file/VCIOKDxHOw7qYkC4ks5Ctp/Adaptive-Story?node-id=1973%3A101053&t=x0pXAz2esWEJ2tWn-1))?

@inikolova The final adaptive rendering is provided in the design and it doesn't include ActionSheet, just "hiding the middle items". If there is a demand for another UI, I've suggested a few new features to the Product managers, that may deserve attention here: https://github.com/telerik/web-components-ux/issues/799#issuecomment-1523702531

File: packages/html/src/breadcrumb/tests/breadcrumb-adaptive.tsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+                                <BreadcrumbItemText text="Item 01"></BreadcrumbItemText>
+                            </BreadcrumbLink>
+                        </BreadcrumbItem>
+                        <BreadcrumbItem>
+                            <Icon className="k-breadcrumb-delimiter-icon" icon="chevron-right" size="xsmall" />
+                        </BreadcrumbItem>
+                        <BreadcrumbItem>
+                            <Icon className="k-breadcrumb-delimiter-icon" icon="chevron-right" size="xsmall" />
+                            <BreadcrumbLink>
+                                <span className="test"></span>
```
Comment (inikolova) [inline packages/html/src/breadcrumb/tests/breadcrumb-adaptive.tsx:63]: Do we need this class?

File: packages/html/src/breadcrumb/tests/breadcrumb-size.tsx [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+                        <BreadcrumbLink root icontext focus>
+                            <Icon icon="home" />
+                            <BreadcrumbItemText text="Home"></BreadcrumbItemText>
+                        </BreadcrumbLink>
+                    </BreadcrumbItem>
+                </BreadcrumbRootItemContainer>
+                <BreadcrumbContainer collapsing="wrap">
+                    <BreadcrumbItem>
+                        <Icon className="k-breadcrumb-delimiter-icon" icon="chevron-right" size="xsmall" />
+                        <BreadcrumbLink focus>
```
[the remark this answers is not in the document]
Comment (magdalenaan) [reply packages/html/src/breadcrumb/tests/breadcrumb-size.tsx:25]: The Breadcrumb component should have a transparent background, so the sizes will not be visible changing the background of the test area.

**Review by inikolova — APPROVED** [review]

**Pull request merged**